data/helpers.d/setting

```diff
@@ -59,6 +59,98 @@ ynh_app_setting_delete() {
     ynh_app_setting "delete" "$app" "$key"
 }
 
+# Add skipped_uris urls into the config
+#
+# usage: ynh_add_skipped_uris [--appid=app] --url=url1,url2 [--regex]
+# | arg: -a, --appid - the application id
+# | arg: -u, --url - the urls to add to the sso for this app
+# | arg: -r, --regex - Use the key 'skipped_regex' instead of 'skipped_uris'
+#
+# An URL set with 'skipped_uris' key will be totally ignored by the SSO,
+# which means that the access will be public and the logged-in user information will not be passed to the app.
+#
+# Requires YunoHost version ?.?.? or higher.
```

Comment:
Will be merged in 3.6.0, so change "?"

Reply:
Are you sure it's going to be ?
This should be done during the testing release.

And could be done already for the previous release.

Reply:
@alexAubin add it to the milestone 3.6.x (but this comment is more a reminder for him)

I can't find these helpers O.o